packages/docs/src/components/content/ContentRenderer.tsx

```diff
@@ -58,7 +58,7 @@ const PreformattedWithLanguageClass = (props: any) => {
 };
 
 const TextWithMaxWidth = (props: any, Component) => {
-  const className = `ds-u-measure--wide ${props.className || ''}`;
+  const className = props.className ? `${props.className}` : null;
```

Comment:
1) Do we need to put this in a template string now? and 2) Do we actually need this ternary or constant at all now?

Reply:
i actually don't think we ever used this. what it was doing was applying `class="ds-u-measure--wide"` to EVERY `<p>` in the site and i didn't see a single instance where props we being used to change that.

Reply:
So could we remove this line entirely is what I was asking